tools/deb/istio-iptables.sh

```diff
@@ -204,6 +204,34 @@ set -o nounset
 set -o pipefail
 set -x # echo on
 
+# Maximum time to wait for an iptables call to succeed when populating the redirection rules.
+# During that time, iptables call will be attempted for every one second.
+IPTABLES_WAIT=30
+# The iptables command path
+IPTABLES_CMD=`which iptables`
+
+# Make sure this function is placed after the iptables calls to cleanup
+# the existing redirection rules and before any iptables call to populate
+# the iptables for traffic redirection.
```

Comment:
I'm not sure why this needs to only be in effect for the redirection rules.  Do the cleanup calls not have the same potential locking issue?

Reply:
You can see from the original script that errors from those calls are ignored. For our use case, the iptables in the network namespace contain default rules only. So cleanup is not required. And if cleanup uses the retry logic, it'd fail  miserably. 

The script may be called after the pod has been up. I see the script has been built into proxyv2 container, etc. Not exactly sure what's it used for. The cleanup logic will be useful in that case.